File: cs/src/AutoDiff/Compiled/Atan2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace AutoDiff.Compiled
{
	class Atan2 : TapeElement
    {
        public int Left;
        public int Right;

        public override void Accept(ITapeVisitor visitor)
        {
            visitor.Visit(this);
        }
	}
}

File: java/src/main/java/AutoDiff/Compiled/Atan2.java
//
// Translated by CS2J (http://www.cs2j.com): 12.11.2016 09:40:32
//

package AutoDiff.Compiled;

public class Atan2  extends TapeElement
{
    public int Left;
    public int Right;
    public void accept(ITapeVisitor visitor) throws Exception {
        visitor.visit(this);
    }

}


